plugins.txt

```diff
@@ -1 +1,4 @@
 pipeline-model-definition:latest
+workflow-support:3.0
```

Comment:
Necessary plugins to execute [the cwp demo](https://github.com/jenkinsci/jenkinsfile-runner/tree/master/demo/cwp) in JDK 11.